plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -93,63 +102,77 @@ open class GenerateGramineBundle @Inject constructor(
         }
     }
 
+    private fun copySeccompToOutputDir() {
+        val seccomp = outputDir.get().asFile.resolve(GRAMINE_SECCOMP).toPath()
+        javaClass.copyResource(GRAMINE_SECCOMP, seccomp)
+    }
+
+    private fun retrieveArchitecture(): String {
+        return linuxExec.execWithOutput(listOf("gcc", "-dumpmachine"))
+    }
```

Comment:
No need to call this as we know what the architecture is inside the container

Reply:
We now call this only when using Python